components/withNavBar.tsx

```diff
@@ -4,6 +4,7 @@ import { useLocale } from 'next-intl';
 import { useTheme } from 'next-themes';
 import type { FC } from 'react';
 
+import { STORYBOOK_MODE_THEME } from '@/.storybook/constants';
```

Comment:
As mentioned before, we shouldn't have Sotrybook constants being imported in Application code, you can create a new constant similar to the Storybook one, on an App constants file, like `next.constants.mjs`